src/resources/ha-style.ts

```diff
@@ -53,8 +53,10 @@ documentContainer.innerHTML = `<custom-style>
       --sidebar-selected-icon-color: var(--primary-color);
 
       /* controls */
-      --toggle-button-color: var(--primary-color);
-      /* --toggle-button-unchecked-color: var(--accent-color); */
```

Comment:
Oh we already had official none paper variables, that should have been used in themes. Should we reuse those instead of creating `switch` ones?

Reply:
We could, but I like switch better 🤷‍♂️

Reply:
That would be breaking, and keeping `toggle-button` in theory not, but since everyone uses `paper` anyway it doesn't really matter I think...

Reply:
Thanks for the great response.  You guys rock!